debian/postinst

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python3 -m pip install python-gnupg==0.5.0 grpcio==1.49.0 pexpect==4.8.0 protobuf==4.21.9 yubikey-manager==4.0.9 cython==3.0.0 stdeb==0.10.0
```

Comment:
To chyba powinny być zależności od innych pakietów Debiana, a nie instalowanie przez PIP. Debian chyba teraz zabrania PIPowania bez wirtualnego środowiska.

https://www.debian.org/doc/debian-policy/ch-relationships.html